opensrp-chw-core/src/test/java/org/smartregister/chw/core/utils/BAJsonFormUtilsTest.java

```diff
@@ -97,6 +107,45 @@ public void getAutoJsonEditMemberFormStringReturnsCorrectJSONObject() throws JSO
         Assert.assertEquals(formTitle, stepOne.getString(TITLE));
     }
 
+    @Test
+    public void getStartFormActivityReturnsCorrectIntent() {
```

Comment:
Kindly follow the test naming convention `test[MethodName][ExpectedResult][Parameters]`